apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/SettingsActivity.java

```diff
@@ -73,6 +73,12 @@
 
     private static boolean sAnalyticsPaused;
 
+    private static String initialStartType;
+
+    private static String oneCollectorLogUrl = "https://mobile.events.data.microsoft.com/OneCollector/1.0";
```

Comment:
Please do not add the URL as a constant in the code. You can find `env.xml` in `res` folder that can contain all variables related to an environment.

Reply:
Yes, resolved in new commit.